docs/fastdds/discovery/static.rst

```diff
@@ -280,3 +280,28 @@ Or you can specify the STATIC EDP configuration content directly.
 |    :end-before: //!                                  |
 |    :dedent: 8                                        |
 +------------------------------------------------------+
+
+Checking STATIC EDP XML Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before Load a static EDP xml file, it would be useful to check its validity and make sure the file would be
+successfully loaded. This verification can be performed on DomainParticipantFactory using check_xml_static_discovery,
```

Comment:
Use a link to the [API reference](https://fast-dds.docs.eprosima.com/en/latest/fastdds/api_reference/dds_pim/domain/domainparticipantfactory.html#_CPPv4N8eprosima7fastdds3dds24DomainParticipantFactory26check_xml_static_discoveryERNSt6stringE). This is usually done defining an alias in [this file](https://github.com/eProsima/Fast-DDS-docs/blob/master/docs/03-exports/aliases-api.include).

Also, DomainParticipantFactory could be a cross-reference to the [corresponding section](https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domainParticipantFactory/domainParticipantFactory.html#domainparticipantfactory)

Reply:
Added in [af7f5b8](https://github.com/eProsima/Fast-DDS-docs/pull/330/commits/af7f5b8485581c6a8ec1bc626569cbdb8e7f9f1d)